submissions/AndriiBabushko/html-css_popup/index.html

```diff
@@ -0,0 +1,186 @@
+                                    <h3>Youtube</h3>
+                                </a>
+                            </li>
+                            <li class="submenu__item">
+                                <a href="#" class="submenu__link">
+                                    <img alt="" class="submenu__img" src="./img/icons/svg/soundcloud.svg"
+                                         title="Soundcloud">
+                                    <h3>Soundcloud</h3>
+                                </a>
+                            </li>
+                            <li class="submenu__item">
+                                <a href="#" class="submenu__link">
+                                    <img alt="" class="submenu__img" src="./img/icons/svg/spotify.svg" title="Spotify">
+                                    <h3>Spotify</h3>
+                                </a>
+                            </li>
+                            <li class="submenu__item">
+                                <a href="#" class="submenu__link">
+                                    <img alt="" class="submenu__img" src="./img/icons/svg/github_alt.svg"
+                                         title="Github">
+                                    <h3>Github</h3>
+                                </a>
+                            </li>
+                            <li class="submenu__item">
+                                <a href="#" class="submenu__link">
+                                    <img alt="" class="submenu__img" src="./img/icons/svg/wikipedia.svg"
+                                         title="Wikipedia">
+                                    <h3>Wikipedia</h3>
+                                </a>
+                            </li>
+                            <li class="submenu__item">
+                                <a href="#" class="submenu__link">
+                                    <img alt="" class="submenu__img" src="./img/icons/svg/behance.svg" title="Behance">
+                                    <h3>Behance</h3>
+                                </a>
+                            </li>
+                            <li class="submenu__item">
+                                <a href="#" class="submenu__link">
+                                    <img alt="" class="submenu__img" src="./img/icons/svg/blip.svg" title="Blip">
+                                    <h3>Blip</h3>
+                                </a>
+                            </li>
+                            <li class="see-more-button">
+                                <input class="see-more-check__input" type="checkbox" name="menu" id="see-more">
+                                <label for="see-more" class="see-more-button__label">
+                                    <img alt="" src="./img/icons/svg/college-see-more.svg" class="see-more-button__img">
+                                </label>
```

Comment:
Add styles on focus state and hide this button when the second part of the popup is opened

Reply:
✓